test/suites/tls_restrictions.sh

```diff
@@ -96,43 +95,240 @@ test_tls_restrictions() {
   # There should now be two volume URLs, one instance, one image, and one profile URL in the used-by list.
   [ "$(lxc_remote project list localhost: --format csv | cut -d, -f9)" = "5" ]
 
-  # Delete resources in project blah so that we can modify project limits.
+  # Delete resources in project blah so that we can modify project features.
   lxc_remote delete localhost:blah-instance --project blah
   lxc_remote storage volume delete "localhost:${pool_name}" blah-volume --project blah
-  test_image_fingerprint="$(lxc_remote image list localhost: --format csv --columns f --project blah)"
   lxc_remote image delete "localhost:${test_image_fingerprint}" --project blah
 
   # Ensure we can create and view resources that are not enabled for the project (e.g. their effective project is
   # the default project).
 
-  # Networks are disabled when projects are created.
+  ### IMAGES (initial value is true for new projects)
+
+  # Unset the images feature (the default is false).
```

Comment:
Isn't the default value `true`? If so, the there are many such comment that would need fixing.

Reply:
No, the default value is false for all project features. The initial value upon creating a project is true for images, storage volumes, storage buckets, and profiles, and false for networks and network zones.

Reply:
At first I thought you were contradicting yourself but then I realized that LXD actually sets those config so unsetting brings a default that is different than the one you get on creation. Quite confusing.

```bash
$ lxc project create blah
Project blah created
$ lxc project show blah
name: blah
description: ""
config:
  features.images: "true"
  features.profiles: "true"
  features.storage.buckets: "true"
  features.storage.volumes: "true"
used_by:
- /1.0/profiles/default?project=blah
$ lxc project unset blah features.images
$ lxc project show blah
name: blah
description: ""
config:
  features.profiles: "true"
  features.storage.buckets: "true"
  features.storage.volumes: "true"
used_by:
- /1.0/profiles/default?project=blah
```

Reply:
These are so-called "create time defaults", similar to https://documentation.ubuntu.com/lxd/en/latest/reference/network_bridge/#network-bridge-network-conf:ipv6.nat